DataCollector/DoctrineDataCollector.php

```diff
@@ -136,6 +137,19 @@ public function collect(Request $request, Response $response, \Exception $except
             }
         }
 
+        // Might be good idea to replicate this block in doctrine bridge so we can drop this from here after some time.
+        // This code is compatible with such change, because cloneVar is supposed to check if input is already cloned.
+        foreach ($this->data['queries'] as &$queries) {
+            foreach ($queries as &$query) {
+                // HttpKernel < 3.2 compatibility layer
+                if (!method_exists($this, 'cloneVar')) {
```

Comment:
Shouldn't both foreach be inside that if ?

Reply:
indeed